index.xml

```diff
@@ -12005,12 +11827,51 @@ interface
+                        href='#dfn-namespace'>namespace</a>, and <var>op</var> is not a <a
+                        class='dfnref' href='#dfn-static-operation'>static operation</a>:
+
+                        <ol>
+                          <li>
+                            If the <span class='esvalue'>this</span> value is <span
+                            class='esvalue'>null</span> or <span class='esvalue'>undefined</span>,
+                            set <var>O</var> to <var>realm</var>'s <a class='dfnref external'
+                            href='https://html.spec.whatwg.org/multipage/webappapis.html#concept-realm-global'>global
+                            object</a>. (This will subsequently cause a <span
+                            class='estype'>TypeError</span> in a few steps, if <var>target</var> is
+                            not the appropriate global object.)
+
+                            <!-- https://www.w3.org/Bugs/Public/show_bug.cgi?id=18547#c9 -->
+                          </li>
+
+                          <li>Otherwise, set <var>O</var> to the <span class='esvalue'>this</span>
+                          value.</li>
+
+                          <li>If <var>O</var> is a <a class='dfnref'
+                          href='#dfn-platform-object'>platform object</a>, then <a class='dfnref'
+                          href='#dfn-perform-a-security-check'>perform a security check</a>,
+                          passing <var>O</var>, <var>realm</var>, <var>id</var>, and "method".</li>
+
+                          <li>If <var>O</var> is not a <a class='dfnref'
+                          href='#dfn-platform-object'>platform object</a> that implements the
+                          interface <var>target</var>, <a href='#ecmascript-throw' class='dfnref
+                          external'>throw</a> a <span class='estype'>TypeError</span>.</li>
+                        </ol>
+                      </li>
+
+                      <li>
                         Let <var>S</var> be the <a class='dfnref'
                         href='#dfn-effective-overload-set'>effective overload set</a> for <a
-                        class='dfnref' href='#dfn-regular-operation'>regular operations</a> with
-                        <a class='dfnref' href='#dfn-identifier'>identifier</a> <var>id</var> on
-                        <a class='dfnref' href='#dfn-namespace'>namespace</a> <var>namespace</var>
-                        and with argument count <var>n</var>.
+                        class='dfnref' href='#dfn-regular-operation'>regular operations</a> (if
+                        <var>op</var> is a regular operation) or for <a class='dfnref'
+                        href='#dfn-static-operation'>static operations</a> (if <var>op</var> is a
+                        static operation) with <a class='dfnref'
+                        href='#dfn-identifier'>identifier</a> <var>id</var> on <var>target</var> and
```

Comment:
Right, and here it's _very_ important that `target` be the actual interface we're defining on and not a mixin, because otherwise you don't have the full overload set.  I really wish we'd split mixins and interfaces into two separate concepts...